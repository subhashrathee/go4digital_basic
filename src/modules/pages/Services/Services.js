import React from 'react'
import Body from '../../common/Body/Body'
import Footer from '../../common/Footer/Footer'
import JSONContent from '../../config/content/services.js'

//Content used to build instagram links
const instagram_link = JSONContent.services_instagram_link.join('');

//Content passed into body component.
var content = JSONContent.services.join('');

//adding our instagram links to content
content +=  "<p>" + 
            instagram_link.replace('VID_ID', 'BZe5g8QjVyw') + 
            instagram_link.replace('VID_ID', 'BcUS3zMDONZ') + 
            instagram_link.replace('VID_ID', 'BZXa8wgjSr7') + 
            "</p>";

/**
* Services page component used to render the page.
*
* Props:      None
* 
* State vars: None
*/
class Services  extends React.Component {
  render () {
    return (
      <div>  
      <div className="hero-image-services"></div>
        <Body heading="Services" content={content}/>
        <Footer />
      </div>
    )
  }
}

export default Services;
