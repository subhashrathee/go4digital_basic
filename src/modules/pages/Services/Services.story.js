import React from 'react';
import { storiesOf } from '@storybook/react';
import { action } from '@storybook/addon-actions';
import { BrowserRouter } from 'react-router-dom'
import Services from './Services'

storiesOf('Pages', module)
  .add('Services', () => (
      <Services />
  ));
