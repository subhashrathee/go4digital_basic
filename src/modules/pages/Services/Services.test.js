import React from 'react';
import Services from './Services';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('Services component', () => {
  const component = renderer.create(
    <Services />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
