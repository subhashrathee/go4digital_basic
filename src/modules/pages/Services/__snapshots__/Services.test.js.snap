// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services component 1`] = `
<div>
  <div
    className="hero-image-services"
  />
  <div
    className="main"
  >
    <h1>
      Services
    </h1>
    <p>
      Our Training comprises a variety and combination of:
    </p>
    <ul>
      <li>
        Practical instruction of Buk Sing philosophies and applications;
      </li>
      <li>
        Strong focus on combat techniques and tactics;
      </li>
      <li>
        Partner drills / Pad work;
      </li>
      <li>
        Sparring;
      </li>
      <li>
        Kung Fu Forms including use of weapons;
      </li>
      <li>
        Wooden dummy training.
      </li>
    </ul>
    <p>
      Sifu Gary Mallia believes that effort equals reward and that the results from his instruction, will improve:
    </p>
    <ul>
      <li>
        Self Confidence and Self Esteem;
      </li>
      <li>
        Ability to comprehensively and effectively defend against a variety of opponents;
      </li>
      <li>
        Physical Fitness and Well Being;
      </li>
      <li>
        Flexibility, Balance and Co-Ordination.
      </li>
    </ul>
    <p>
      <span
        style={
          Object {
            "paddingLeft": "10px",
          }
        }
      >
        <iframe
          allowtransperency="true"
          frameBorder="0"
          height="290"
          scrolling="no"
          src="//instagram.com/p/BZe5g8QjVyw/embed"
          width="230"
        />
      </span>
      <span
        style={
          Object {
            "paddingLeft": "10px",
          }
        }
      >
        <iframe
          allowtransperency="true"
          frameBorder="0"
          height="290"
          scrolling="no"
          src="//instagram.com/p/BcUS3zMDONZ/embed"
          width="230"
        />
      </span>
      <span
        style={
          Object {
            "paddingLeft": "10px",
          }
        }
      >
        <iframe
          allowtransperency="true"
          frameBorder="0"
          height="290"
          scrolling="no"
          src="//instagram.com/p/BZXa8wgjSr7/embed"
          width="230"
        />
      </span>
    </p>
  </div>
  <footer>
    <p>
      Â© 
      2018
       go4digital. All rights reserved.
    </p>
  </footer>
</div>
`;
